diff --git a/node_modules/cose-js/lib/mac.js b/node_modules/cose-js/lib/mac.js
index 3a86e0c..9ab017e 100644
--- a/node_modules/cose-js/lib/mac.js
+++ b/node_modules/cose-js/lib/mac.js
@@ -2,7 +2,7 @@
 /* jslint node: true */
 'use strict';
 
-const cbor = require('cbor');
+const cbor = require('cbor-web');
 const aesCbcMac = require('aes-cbc-mac');
 const crypto = require('crypto');
 const Promise = require('any-promise');
